russian/thinking-about/pen-paper/pen-paper.md

```diff
@@ -0,0 +1,11 @@
+Ты развернул бумажку и прочитал:
+
+"
+Когда ты чувствуешь, что тебе трудно встать утром, подумай про себя: я пробудился для исполнения обязанностей человека, почему же огорчаться тем, что мне нужно сделать то, для чего я родился, для чего я послам, в мир? Разве я родился затем, чтобы удобно лежать под одеялом?— Но это доставляет такое удовольствие. — А разве ты родился для удовольствия, а не для того, чтобы действовать и работать? Разве ты не видишь, что растения, муравьи, пауки, каждый исполняют свои обязанности и все служат мировой гармоний? А ты будешь отказываться исполнять свою обязанность человека, ты не хочешь делать того, что сообразно твоей природе. — Но нужно же отдохнуть. — Ты хочешь отдыхать? Но ведь природа положила предел этой потребности так же, как потребности есть и пить. А ты теперь переходишь этот предел, ты идешь дальше того, что должно бы тебя удовлетворить; а вот в деятельности это не совсем так: ты остаешься ниже возможного. Ты не любишь сам себя, если не любишь свою природу и ее требования. Да, те, которые любят свое дело, сохнуть над работой, забывая и пить и есть; а ты на свою природу обращаешь меньше внимания, чем резчик на свое искусство, танцор на танцы, скупец на деньги, а безумный честолюбец на свою славу. Когда они работают, они меньше заботятся об еде или об сне, чем о привлекающем их успехе своего дела. Неужели действия, имеющие целью общественное благо, кажутся тебе менее важными и менее достойными твоих забот? 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.